apps/system/utils/kdbg_heapinfo.c

```diff
@@ -120,10 +120,10 @@ static void heapinfo_print_values(char *buf)
 	int i;
 	stat_data stat_info[PROC_STAT_MAX];
 
-	stat_info[0] = strtok(buf, " ");
+	stat_info[0] = buf;
 
-	for (i = 1; i < PROC_STAT_MAX; i++) {
-		stat_info[i] = strtok(NULL, " ");
+	for (i = 0; i < PROC_STAT_MAX - 1; i++) {
+		stat_info[i] = strtok_r(stat_info[i], " ", &stat_info[i + 1]);
```

Comment:
The index of proc data is always PROC_STAT_MAX??

Reply:
Yes, the count of proc data is PROC_STAT_MAX.
So index is in (0~PROC_STAT_MAX-1)